src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
It is not used for ARM / ARM64 in CoreCLR, the information is never encoded using the compact encoding.